go/api/go_api/session.py

```diff
@@ -12,7 +13,8 @@ class SessionStore(SessionBase):
     """
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
-        self.session_manager = vumi_api().session_manager
+        self.session_manager = SessionManager(
+            get_redis_manager().sub_manager('session_manager'))
```

Comment:
This and doesn't actually solve the underlying problem, which is that we only need one small thing but the standard way of getting it is to create a big container full of all sorts of things.

In the case of this PR, we'd need to know that we can safely clean up the Riak manager that gets created but not the Redis manager. In the case of the `SessionStore` we also have nowhere to do any cleanup that might be necessary when we're finished with the session store -- we're already potentially leaking Redis clients with this code.